packages/frontend/src/creation-form/constraint-values-form/ranged-values.tsx

```diff
@@ -43,15 +43,24 @@ export const RangedValuesConstraintForm = ({
         let value0ErrorText = "";
         let value1ErrorText = "";
 
-        if (!value0) {
+        if (
+            type &&
+            type.value === ConstraintType.BETWEEN &&
```

Comment:
Shouldn't it be `Range` and `NotBetween` too?

Reply:
Fixed in 53e3ec4fb2bd31d742d3ea3941251bece6547356.